src/backend/cdb/cdbpartition.c

```diff
@@ -7041,8 +7041,8 @@ atpxPartAddList(Relation rel,
 
 				skipTableRelid = RangeVarGetRelid(t->relation, NoLock, true);
 			}
-		}
 
+		/* FIXME: indent this or remove condition above if it is not needed */
```

Comment:
```suggestion
		/* FIXME: indent this */
```
I think, that condition above is needed, because it deals with first list item, while next cycle starts with second